en/docs/security-announcements/security-advisories/2020/WSO2-2020-0699.md

```diff
@@ -21,24 +21,23 @@ cvss: "7.6 (CVSS:3.0/AV:N/AC:L/PR:L/UI:R/S:U/C:H/I:H/A:L)"
 
 
 ### OVERVIEW
-A potential security vulnerability has been identified in SMS OTP Authenticatior.
+A potential security vulnerability has been identified in SMS OTP Authentication.
```

Comment:
I think, the word should be `authenticator`. WDYT?

Reply:
will fix. thanks!

Reply:
Fixed in 27cf1e388a80e0db2fcda559634eacab8feb3e4a